packages/create/templates/hello-typeorm/.vscode/tasks.json

```diff
@@ -0,0 +1,19 @@
+{
+    // Configure the build command for DBOS projects using VSCode.
+    // Documentation on tasks.json: https://code.visualstudio.com/docs/editor/tasks
+    "version": "2.0.0",
+    "tasks": [
+      {
+        "label": "npm: build",
```

Comment:
Do we also want to setup nodemon watcher tasks? @chuck-dbos knows more about how that works

Reply:
I didn't run it but it looked like they are already set up.